test/integration/webhook/jobset_webhook_test.go

```diff
@@ -130,5 +130,16 @@ var _ = ginkgo.Describe("jobset webhook defaulting", func() {
 				return js.Spec.ReplicatedJobs[0].Network != nil && *js.Spec.ReplicatedJobs[0].Network.EnableDNSHostnames
 			},
 		}),
+		ginkgo.Entry("validate enableDNSHostnames can't be set if job is not Indexed", &testCase{
```

Comment:
this place is the reason why test failed, however I'm not familiar with integration test and this testcase might have error.. I have no clues now..

Reply:
all tests are passing now so looks like you figured it out